### DISCUSSION TITLE
Current Agent Version?

### DISCUSSION THREAD

**Discussion by sysopdm** [Ylianst/MeshCentral#4754] [Q&A]
Hello all,

   I recently updated my MeshCentral server to version 1.0.97 and was expecting all my Mesh agents to update.

They haven't updated and still showing the version from August 25.  Is this correct or am I having an updating issue with my agents?

Thanks!

DM

**Comment by PathfinderNetworks** [comment]
Version 143332014 dated August 25, 2022 is the current version.

**Reply by sysopdm** [reply]
Great and thanks!
